portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,370 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+
+/** SHARED FOR NAV BAR */
+ul{
+    list-style: none;
+}
+
+a{
+    text-decoration: none;
+}
+
+nav{
+    background-color: black;
+}
+.navbar-logo{
+    pointer-events: none;
+    padding-left: 50px;
+    padding-right: 50px;
+    font-weight: bold;
+    color: white;
+}
+
+.navbar-logo:hover{
+    color: white;
+}
+
+.nav li a{
+    padding-left: 50px;
+    padding-right: 50px;
+    color: white;
+ }
+
+nav .nav a:hover{
+    text-decoration: underline white;
+    transition: 0.5s; 
+  }
+
+                                                        /**INDEX.HTML*/
+
+.index-body{
+   
+    font-family: 'Segoe UI', Tahoma;
+    scroll-behavior: smooth;
+   background-color: #FFFFFF;
+    background-size: cover;
+
+}
+
+.index-picture{
+width: fit-content;
+float: left;
+}
+
+.index-img{
+    width: 550px;
+    height: 500px;
+    padding-top: 50px;
+    padding-left: 50px;
+}
+
+.index-flex-container{
+    display: flex;
+    flex-direction: column;
+}
+
+.index-main-intro{
+    float: right;
+    text-align: center;
+    width: 600px;
+    padding-top: 100px;
+    padding-right: 100px;
+   right: 200px;
+}
+
+                                                            /**GALLERY.HTML*/
```

Nit: I know Gabriel already commented about commenting `.css` files, and I don't know if it's me or GitHub, but some comments you made appear to be "centered" for some reason. I think it's better if it's just at the beginning like the rest of the lines.

```suggestion
 /**GALLERY.HTML*/
```

---

Thanks for the feedback Sebastian! I changed the comments in the style.css based on both suggestions.